app/api/emails.py

```
@@ -0,0 +1,191 @@
+    This project currently treats `X-Caller-Id` as an authenticated identity
+    provided by a trusted upstream (e.g., API gateway/service mesh). For
+    internet-facing deployments, replace this with verified authentication
+    (JWT/API key) and derive the caller id from that identity.
+    """
+    return x_caller_id
+
+
+@router.post("/", response_model=EmailResponse, status_code=status.HTTP_202_ACCEPTED)
+async def send_email(
+    email_request: EmailRequest,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)